app/src/main/res/layout-land/topic_practice_header_view.xml

```diff
@@ -12,12 +12,10 @@
   <androidx.constraintlayout.widget.ConstraintLayout
     android:layout_width="match_parent"
     android:layout_height="wrap_content"
-    android:layout_marginStart="32dp"
     android:layout_marginTop="32dp"
-    android:layout_marginEnd="24dp"
-    android:layout_marginBottom="8dp"
-    android:paddingStart="48dp"
-    android:paddingEnd="48dp">
+    android:layout_marginLeft="96dp"
```

Comment:
Use marginStart and marginEnd to support RTL here and elsewhere.

Reply:
Changed